### PR TITLE
Reformatted slpmap documentation

### PR DESCRIPTION
**Pull request by kaitlyndlee**
## Description

The way slpmap's documentation was formatted was not clear about the use of the PIXRES parameter. In order to make the use of the parameter more clear, we moved around a few paragraphs and highlighted a few points. Also fixed a few typos.

## Related Issue

https://github.com/USGS-Astrogeology/ISIS3/issues/3562

## Motivation and Context

Clear up confusions about PIXRES parameter.

## How Has This Been Tested?

Generated docs and viewed them in a web browser.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Documentation change (update to the documentation; no code change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read and agree to abide by the [Code of Conduct](https://usgs-astrogeology.github.io/code/)
- [x] I have read the [**CONTRIBUTING**](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CONTRIBUTING.md) document.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

## Licensing
This project is mostly composed of free and unencumbered software released into the public domain, and we are unlikely to accept contributions that are not also released into the public domain. Somewhere near the top of each file should have these words:

> This work is free and unencumbered software released into the public domain. In jurisdictions that recognize copyright laws, the author or authors of this software dedicate any and all copyright interest in the software to the public domain.

- [x] I dedicate any and all copyright interest in this software to the public domain. I make this dedication for the benefit of the public at large and to the detriment of my heirs and successors. I intend this dedication to be an overt act of relinquishment in perpetuity of all present and future rights to this software under copyright law.

### CODE REVIEW

**Review by Kelvinrr — COMMENTED** [review]
It looks well enough to me but I'd rather get a review from someone who knows more about this application. @scsides @jessemapel @kberryUSGS?

File: isis/src/base/apps/slpmap/slpmap.xml [hunk 1]
```diff
@@ -50,61 +42,63 @@
...
-    </p>
-    <p>
-      The default setting PIXRES=AUTOMATIC will not work. You will need to set PIXRES=FILE
-      (recommended for a map projected file) or  PIXRES=USER. Note: unlike PIXRES=AUTOMATIC,
-      these later two methods do not correct for distortions int he map projection. They will
-      return good results for smaller regious when the map projection is defined to minimixe
-      distortions (e.g. LROC NAC or HIRISE sereo DTMs).
+      The slope equation above assumes the pixels are not square, hence the two variables:
+      X_PIXEL_RESOLUTION and Y_PIXEL_RESOLUTION. That is, the x distance across the pixel is not 
+      equal to the y distance form top to bottom of the pixel.
```
Comment (krlberry) [inline isis/src/base/apps/slpmap/slpmap.xml]: typo: form -> from

**Review by krlberry — CHANGES_REQUESTED** [review]
I think I found a single typo, but other than that, looks good to me!

**Comment by jessemapel** [comment]
[Enable slpmap to accept cubes made with elevation instead of only radius information](https://trello.com/c/wMl5PtOv/86-enable-slpmap-to-accept-cubes-made-with-elevation-instead-of-only-radius-information)

**Review by jessemapel — COMMENTED** [review]
This looks good pending Kristin's typo fix. It would be nice to re-do this using latex and properly defining kernel matrices, but that's beyond the scope of this PR>

**Review by krlberry — APPROVED** [review]
:+1: I forgot that I could just quickly fix the typo I found myself.

**Pull request merged**